src/main/java/nextstep/ladder/domain/line/Lines.java

```diff
@@ -0,0 +1,38 @@
+package nextstep.ladder.domain.line;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import nextstep.ladder.error.exception.LadderHeightSizeException;
+
+public class Lines {
+
+    private List<Line> lines;
+
+    public Lines(int height, int countOfPerson) {
+        this(toLines(height, countOfPerson));
+    }
+
+    public Lines(List<Line> lines) {
+        this.lines = lines;
+    }
+
+    private static List<Line> toLines(int height, int countOfPerson) {
+        if (height < 1){
+            throw new LadderHeightSizeException(height);
+        }
+
+        return IntStream.range(0, height)
+            .mapToObj(currentHeight -> new Line(countOfPerson))
+            .collect(Collectors.toList());
```

Comment:
Stream.generate() 를 활용해 보셔도 좋을 것 같아요
```suggestion
        return Stream.generate(() -> new Line(countOfPerson))
                .limit(height)
                .collect(Collectors.toList());
```

Reply:
엇 이런게 있군요
항상 반복문을 IntStream으로만 바꿔서 사용했는데 이번에 좋은 문법 배워가네요
피드백 반영하겠습니다